src/integrationtests/java/com/aws/greengrass/integrationtests/e2e/util/IotJobsUtils.java

```diff
@@ -110,8 +131,13 @@ public static void cancelJob(IotClient client, String jobId) {
      * @return String formatted ARN
      */
     public static String generateMockConfigurationArn(String resourceIdVersion) {
-        return Arn.builder().partition("aws").accountId("1234567890").region("test-region")
-                .service("gg").resource(String.format("configuration:%s", resourceIdVersion)).build()
+        return Arn.builder()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential log injection detected. Ensure all untrusted input is properly sanitized before logging. Use parameterized logging or validate input against a allow list to prevent log injection vulnerabilities. Consider using a dedicated logging library&#39;s built-in sanitization features when available. Learn more - https://cwe.mitre.org/data/definitions/117.html